src/marshmallow_sqlalchemy/fields.py

```diff
@@ -41,9 +42,16 @@ class Related(fields.Field):
         "expected a dictionary with keys {keys!r}"
     }
 
-    def __init__(self, column=None, **kwargs):
+    def __init__(self, columns=None, column=None, **kwargs):
+        if column is not None:
+            warnings.warn(
+                "`column` parameter is deprecated and will be removed in future releases. "
+                "Use `columns` instead.",
+                DeprecationWarning,
+            )
+            columns = column
```

Comment:
Maybe only use column if columns is None.

In other words, if user provides both, use columns. Keep warning.

```py
        if column is not None:
            warnings.warn(
                "`column` parameter is deprecated and will be removed in future releases. "
                "Use `columns` instead.",
                DeprecationWarning,
            )
            if columns is None:
                columns = column
```

Reply:
sure @lafrech , should i also add message informing overriding the columns value with column as warning message ?
`'column' parameter is deprecated and will be removed in future releases.Use 'columns' instead. setting columns = column`
so that user is informed about what value is used?

Reply:
Don't bother. If user sets both, they must know about columns. They may even use both with the same value for backward compat or something. In fact, it shouldn't happen, really. Just saying in this case, use columns. And keep the warning to remind about the deprecation variable.

Reply:
ok sure, updating changes